SparkPost/2web2ui
# FE-1273 Fix Chart tooltips in Report Builder / Dashboard v2

### What Changed
- Moves the report builder / dashboard v2 chart tooltip into the shared components folder

### How To Test
- Verify tooltip works on the new dashboard
- Verify tooltip works on reportbuilder
- Verify tooltip works on usage RV chart

### To Do

  > Manage reviewers' expectations on missing features, tests, bugs, late design changes, ...)
